addons/point_of_sale/static/src/app/navbar/cash_move_popup/cash_move_popup.js

```diff
@@ -30,14 +30,16 @@ export class CashMovePopup extends Component {
     }
 
     async confirm() {
-        const amount = parseFloat(this.state.amount);
+        let amount = Number(this.state.amount);
```

Comment:
Why use `Number`? It's possible that the local of the user has different `decimal` symbol, for example, Belgium local uses `,` as decimal. You should use the parser that's compatible to the local.

Reply:
Good catch